aldryn_forms/boilerplates/bootstrap3/templates/aldryn_forms/emails/notification.subject.txt

```diff
@@ -1 +1,3 @@
-[Form submission] {{ form_name }} 
\ No newline at end of file
+{% with subject=form_plugin.email_notification_subject %}
+{% if subject %}{{ subject }}{% else %}[Form submission] {{ form_name }}{% endif %}
+{% endwith %}
```

Comment:
Remember that subject has to be a one-liner(no trailing newline even) if plain Django Email functions will be used.